.cz-config.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
 		{ value: "feat", name: "feat:		A new feature" },
 		{ value: "fix", name: "fix:		Bugs, typos, etc" },
 		{ value: "misc", name: "misc:		Other formats like tweaks and such" },
-		{ value: "tests", name: "tests:	Tests, jest, binTestCases, etc" }
+		{ value: "tests", name: "tests:	Tests, jest, binTestCases, etc" },
+		{ value: "break", name: "break: changes that break the behaviour of the cli" }
```

Comment:
Can you do asc ordering of the new value?
https://github.com/webpack/webpack-cli/blob/8a90ea4d3b9c8a1d1c72fc78399852c296266717/.cz-config.js#L19